h2/src/main/org/h2/engine/DbSettings.java

```diff
@@ -334,6 +334,13 @@
      */
     public final boolean compressData = get("COMPRESS", false);
 
+    /**
+     * Database setting <code>IGNORE_CATALOGS</code>
+     * (default: false).<br />
+     * Compress data when storing.
```

Comment:
This description is for another setting.

Reply:
fixed